abkayit/training/tutils.py

```diff
@@ -350,6 +350,9 @@ def cancel_all_prefs(trainess, cancelnote, site, ruser, d):
                     context['recipientlist'].extend(approvedpref[0].course.authorized_trainer.all().values_list(
                         'user__username', flat=True))
             send_email_by_operation_name(context, "notice_for_canceled_prefs")
+            recontext = context.copy()
+            recontext['recipientlist'] = [ruser.email]
+            send_email_by_operation_name(recontext, "notice_for_canceled_prefs")
```

Comment:
@Cediddi aynı e-postayı öğrenciye de göndereceksek context'in copyasını almaya gerek var mı?
context['recipientlist']'e katilimcinin e-postasını eklesek daha iyi olur.
bu arada iptal işlemini her zaman öğrencinin kendisi yapmıyor. yönetici veya görevli de yapabiliyor. O sebeple ruser'ın emailini eklemek yerine trainess.user.email'i eklemek gerekiyor. (ruser: request user -  http istegini yapan kullanıcı.)